test/src/state.js

```diff
@@ -16,8 +18,9 @@ const defaultState = {
   exampleRecordings: fs.existsSync("./test/example-recordings.json")
     ? JSON.parse(fs.readFileSync("./test/example-recordings.json"))
     : {},
-  headless: false,
+  headless: !!process.env.RECORD_REPLAY_PLAYWRIGHT_HEADLESS,
```

Comment:
Do we add this to the `.env` file, for better visibility that this option even exists, since we're already exposing it so it can be parametrised?

Reply:
It's set in `.github/actions/test/action.yml` and I sometimes set it in `.vscode/settings.json` if I want to run the tests locally in headless mode.